src/client.cpp

```diff
@@ -626,12 +618,16 @@ QString CClient::SetSndCrdDev ( const QString strNewDev )
         Sound.Start();
     }
 
-    // in case of an error inform the GUI about it
     if ( !strError.isEmpty() )
     {
-        emit SoundDeviceChanged ( strError );
+        // due to error, disconnect
+        Disconnect();
```

Comment:
Why hide information like this?  Is it any way related to the rest of the PR?

Reply:
Should probably be reverted...